docs/source/usage/param/particles/current.rst

```diff
@@ -0,0 +1,64 @@
+.. _usage-params-core-currentdeposition:
+
+Current Deposition
+""""""""""""""""""
+
+The current solver can be set in :ref:`species.param <usage-params-core>` or directly per species :ref:`speciesDefinition.param <usage-params-core>`.
+
+The following operations can be applied in the ``picongpu::particles::InitPipeline`` of the latter:
+
+.. _usage-params-core-particles-currentsolver:
+
+Current Solver
+''''''''''''''
+
+Esirkepov
+~~~~~~~~~
+
+.. doxygenstruct:: picongpu::currentSolver::Esirkepov
+   :project: PIConGPU
+
+EmZ
+~~~
+
+.. doxygenstruct:: picongpu::currentSolver::EmZ
+   :project: PIConGPU
+
+VillaBune
+~~~~~~~~~
+
+.. doxygenstruct:: picongpu::currentSolver::VillaBune
+   :project: PIConGPU
+
+EsirkepovNative
+~~~~~~~~~~~~~~~
+
+.. doxygenstruct:: picongpu::currentSolver::EsirkepovNative
+   :project: PIConGPU
+
+
+.. _usage-params-core-particles-depositionstrategy:
+
+Deposition Strategy
+'''''''''''''''''''
+
+A current solver supports a strategy to change how the algorithm behaves on different compute architectures.
+The strategy is optional, changing could could effect the performance negative.
```

Comment:
Double `could`, also changing can affect performance in both positive and negative ways I think? So perhaps this sentence should be smth like `The strategy is optional, could affect performance`. Perhaps also add "...; the default value should normally be optimal"